src/pages/en/tutorial/6-islands/3.md

```diff
@@ -8,8 +8,43 @@ setup: |
   import Checklist from '~/components/Checklist.astro';
   import PreCheck from '~/components/tutorial/PreCheck.astro';
 ---
+There's one more edit to make...
 
-Now you have your very own Astro blog!
+```astro title="src/pages/about.astro" del={5} ins={6}
+---
+import BaseLayout from "../layouts/BaseLayout.astro";
+const pageTitle = "About Me";
+const happy = true;
+const finished = false;
+const finished = true;
```

Comment:
Wow, love it! It’s almost like you planned this!

Reply:
(I guess requires an update to the completed project?  😅 )